stdlib/LinearAlgebra/test/bidiag.jl

```diff
@@ -44,6 +44,7 @@ Random.seed!(1)
         end
         @test eltype(Bidiagonal{elty}([1,2,3,4], [1.0f0,2.0f0,3.0f0], :U)) == elty
         @test eltype(Bidiagonal([1,2,3,4], [1.0f0,2.0f0,3.0f0], :U)) == Float32 # promotion test
+        @test isa(Bidiagonal(Float64[], Float64[], :U), Bidiagonal)
```

Comment:
You could strengthen the test further (otherwise you'll do the exact same test for every `elty`):
```suggestion
        @test isa(Bidiagonal(elty[], elty[], :U), Bidiagonal{elty,Vector{elty}})
```

Reply:
Yes, this looks much better to be honest.